src/portable/synopsys/dwc2/dwc2_esp32.h

```diff
@@ -34,10 +34,22 @@
 
 #include "esp_intr_alloc.h"
 #include "soc/periph_defs.h"
-//#include "soc/usb_periph.h"
-
-#define DWC2_REG_BASE       0x60080000UL
-#define DWC2_EP_MAX         6             // USB_OUT_EP_NUM. TODO ESP32Sx only has 5 tx fifo (5 endpoint IN)
+// #include "soc/usb_periph.h"
+
+#if TU_CHECK_MCU(OPT_MCU_ESP32P4)
+  // TODO: ESP32P4 has two USB: USB20 (HS) and USB11 (FS). Make it configurable.
+  #define DWC2_USB20_BASE     0x50000000UL
+  #define DWC2_USB11_BASE     0x50004000UL
+  #define DWC2_REG_BASE       DWC2_USB20_BASE
```

Comment:
please check `dwc2_stm32.h` to see how we can support more USB peripherals from TinyUSB